draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1147,15 +1147,61 @@ Decoy digests increase the size of the SD-JWT. The number of decoy digests (or w
 
 ## Unlinkability
 
-Colluding Issuer/Verifier or Verifier/Verifier pairs could link issuance/presentation
-or two presentation sessions to the same user on the basis of unique values encoded in the SD-JWT
-(Issuer signature, salts, digests, etc.).
-
-To prevent these types of linkability, various methods, including but not limited to the following ones can be used:
-
-- Use advanced cryptographic schemes, outside the scope of this specification.
-- Issue a batch of SD-JWTs to the Holder to enable the Holder to use a unique SD-JWT per Verifier. This only helps with Verifier/Verifier unlinkability.
-
+Unlinkability is a property that prevents an adversary from correlating
+credential presentations of the same user (beyond the user's consent).
```

Comment:
This sentence doesn't feel quite right/natural to me. Though I'm not sure this suggestion is particularly great either: 

```suggestion
Unlinkability is a property whereby adversaries are prevented from correlating
credential presentations of the same user (beyond the user's consent).
```